app/src/main/java/com/github/houseorganizer/houseorganizer/house/HouseSelectionActivity.java

```diff
@@ -78,7 +77,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void requestPermission() {
         if (ActivityCompat.checkSelfPermission(this, Manifest.permission.ACCESS_FINE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
-            // Comment next line to pass the tests, GrantPermissionRule doesn't prevent the pop up from appearomment next line to pass the tests, GrantPermissionRule doesn't prevent the pop up from appear
+            // Comment next line to pass the tests, GrantPermissionRule doesn't prevent the pop up from appearing
             // TODO: Find alternative
             //requestPermissions(new String[] {Manifest.permission.ACCESS_FINE_LOCATION}, PERMISSIONS_FINE_LOCATION);
         }
```

Comment:
is leaving this commented out fine ? does it just assume it has permission or how does it work since we dont request it

Reply:
I did not touch this part that was done by Sandra last sprint
I just changed the comment because it was repeated
This is used for adding the current house location when creating one but it is not functionnal yet

Reply:
oh okay no problem then